docs/events/timeline-ui-overview.asciidoc

```diff
@@ -110,20 +98,20 @@ Filter for field present::
 Converts a `field with value` filter to a `field exists` filter.
 
 NOTE: When you convert a <<timeline-templates-ui, Timeline template>> to a
-Timeline, some fields may be disabled. For more information, see
+Timeline, some fields may be disabled. For more information, refer to
 <<template-legend-ui>>.
 
 [discrete]
 [[timeline-to-cases-ui]]
 == Attach Timeline to a case
 
 To attach a Timeline to a new or existing case, open it, click *Attach to case* in the upper
-right corner, then select one of these:
+right corner, then select either:
 
-* Attach to new case
-* Attach to existing case...
+* *Attach to new case*; or,
+* *Attach to existing case*.
```

Comment:
I'm not sure a lot of what's changed here is needed. We don't typically write bulleted lists as if they're a continuous, grammatically complete sentence. But it could just be a regular sentence; a two-item bullet list is kind of over-structured anyway. So:

> To attach a Timeline to a new or existing case, open it, click \*Attach to case\* in the upper right corner, then select either \*Attach to new case\* or \*Attach to existing case\*.

Reply:
👍 👍